gmaps/marker.py

```diff
@@ -161,22 +165,26 @@ def _symbol_layer_options(
     return _merge_option_dicts(options)
 
 
-def _marker_layer_options(locations, hover_text, label):
+def _marker_layer_options(locations, hover_text, label, info_html):
     number_markers = len(locations)
     if _is_atomic(hover_text):
         hover_text = [hover_text] * number_markers
+    if _is_atomic(info_html):
+        info_html = [info_html] * number_markers
     if _is_atomic(label):
         label = [label] * number_markers
     options = {
         "location": locations,
         "hover_text": hover_text,
+        "info_html": info_html,
         "label": label
     }
     return _merge_option_dicts(options)
 
 
 def symbol_layer(
-        locations, hover_text="", fill_color=None, fill_opacity=1.0,
+        locations, hover_text="", info_html='', fill_color=None,
```

Comment:
Could we change the single quotes to double quotes (to be consistent with the rest of the code)?